src/Window.cpp

```diff
@@ -834,9 +820,7 @@ void Window::startRun() {
         m_resetButton->setEnabled(true);
 
         // reset score
-        stats->reset(TOTAL_DISTANCE);
-        stats->reset(TOTAL_TURNS);
-        stats->reset(EFFECTIVE_DISTANCE);
+        stats->resetAll();
```

Comment:
Nitpick: Do you mind moving this directly below the line that clears the run output, so that it's obvious that we always reset the stats whenever the run output is cleared? Note that this is a slight behavior change. If the next mouse algorithm fails to start, the stats will still be cleared. That should be ok though, right?